common/persistence/persistence_interface.go

```diff
@@ -186,6 +186,16 @@ type (
 		GetDLQAckLevels(ctx context.Context) (*InternalQueueMetadata, error)
 	}
 
+	//
```

Comment:
Can you add `TODO: document me` or something similar to ensure we don't leave all of these doc strings empty?